frontend/src/static/js/v_summary.js

```diff
@@ -251,6 +251,19 @@ window.vSummary = {
       return res;
     },
 
+    getRepoLink(repo) {
+      const { REPOS } = window;
+      const urlString = repo.location;
+
+      if (urlString.includes('://github.com')) {
+        return `https://github.com/${
+          REPOS[repo.repoId].location.organization}/${
```

Comment:
the original indentation is actually correct; line wrap should have double indent